streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBWindowStoreTest.java

```diff
@@ -115,7 +127,7 @@ public void closeStore() {
     @SuppressWarnings("unchecked")
     @Test
     public void shouldOnlyIterateOpenSegments() {
-        windowStore = createWindowStore(context, false, true);
+        windowStore = createWindowStore(context);
```

Comment:
old code sets `retainDuplicates=true` but new code sets `retainDuplicates=false`

Reply:
Can we remove the `createWindowStore(ProcessorContext)` and just call the one with 2 params?